datasets/modis/modis-vegetation-example.ipynb

```diff
@@ -0,0 +1,776 @@
w88spUSpj6hEv9Nb4myd+i1Wzzy9vPMRqZVRsXE93TjMJDNarfWwt4tXRMntBlf924Xf4970n+I7qG8itjIY6pR+XeGO6yOXRIuPQYDSo8V33Xeaau8B6qce/7D7FoVfmuxqX0coxv3zwJi6297jea/HfnX6aDx08JHDaUkrTFP6ZX+s8QopEqzQjiFUeqO7QCcv0Y4er/TbzzpSzi4f0Q5tpZND1HBadMc8drFEyAu6M5zhe6SFLGb2wxFsqt1gwxtyYzjMMLaJUoW64yFJK33eoGR5ta8ztSRNNSQSSV0nYntRENo2c8N72ZVrqhJt+m0V9yCuDC3zH/FVa6oSyFnB7MsezszXOzx0KQpwxQZFSmtqEIFVxEz33WqRsh3Xa2phRYtPWRuhERRPpbtAsCuej4slNxWPr+owNv5njm8UatxfVWNO71FWZRW3Ir91jVz/JZIJU4fZkjlMVcQiPU0WEE0ciiHTRHvPR/YvMmSJ+wlQiSk5A05hxY9xia1AjSE6zUhKo4jcGCyw5I0aRKbK+5DhH38vcGjVZKQ3F+hVpBamvbIrJWBgruKnIT3ts8S5f3DmOrsYMZkJ+JIiMYBgxH92+wPeuCsn2xzbO45ghRh5o/fGd80SJTNkIeTVcomL67FHlnQtXuRm0ebi+RVRTeLazDoClRjxevc0gdjgMy4w8k5/ZejsNw8VQYk7oHcqyzye69/GqusbdsMkfXf4SH+k+QD+weWfrOq9NlqkZHqqc8qHuwzxc3+Jm0ObXNh/ksflNLCVkEFq8c+EGz/bWBQjGnBFnCodemWV7xCc2z/HY4l0+t3+S840D7qvs8cpoGUcLeF/rNT508BA7w7lCct6ZOUUe17naIZf7Qko1C/RiAhVEKlkmcbre4WJ5l08enBMo9mmVB+eEP9lQReDvNNCx9Yj9foWluRH7swpte0rHE0GsJ6uCQDwJDNJU5rXBElXd59G6ODy+PlnCjXX23AoXG/sMQgEymMY6JVUckINUoaZ79APRaDxqYhlyzLnSPh+7x3X4SGrXlUqUFJEr5SYG+2GFxJIYxTZV1WUzbDKOTeqayzASPpyHa1tc1+Z57XCJa/0W71q+UcQEiNeoxTCy0KQUL9EpKQEvjtdy8I3L1VmbOFMExj/RWLMGvJHL2SqaUvgEZSnjeKVHzfAoaUGxzr8wO877l15iFNv82sHDzBkulhKyoI/56TvvIs0kTlS7/NnFp/lXB08B8GP1ZxmmNo85tzhv7vLBzsPsz4QS4zuaV5kmJr++cwlTjfOGmsVnRyd509ImZ5wDABaWhBrnV24/xGpNvB5TQxKRJrqLLif8ne3v4wdbL3LLbOLGOuedXZ6JTrIXVjnwyqyX+tRUl/X5Hm6qs+9XOF/ZZxhZrNoDtlxR8J+wOsW6dbZ0wDg2C0jPUbYiICSNvsioa+kTSkogvMhyxEvdZUxVRH/8yMqL+JmKKcX8Wu8hLtb3WNDHTGKTu26DOKdT7k4rrFf7lNWAsiZUT6qcUlZ8epGDo4rnYTuo4ygiPuddjatshw3GsXjOLlV2GcQ2vcihrommpiYnuLHOqXKHWWwU4JR+XGKaiGbvvUJU/tCv/7Jofl99nfha7/yGu6QkSX8J+BNAF/hZ4L/PsiySJEkGbgDfMsWUIccct0Xm0HPjY1xTF3iodJeS4rPhNzluiGIjylQcOeABe5O2NqIbCzrYh+P7eWBph9vDOd4YL3C+vYsipQwTmxN6h0lqcttrMtJsjuldhomNKUfMUiM38sc01Qmj2BJZJzkpzU9Mbs+alDX/d5F7xr7BTVcgNGUp4+qsjSyl7MxqJKkoUtxUx0105v+/3P13lGVpetaJ/rY3Zx9/wpuM9K6yfHW1r2rfLddCCNFISDDMoMGIhdEMiMvcuQbG3bm4uUgIIXRHAmRBSEJqtffV1eWyq7LS28jwEce77c388e3Y1a3uVnchBLS+tXKtzMgTcSLi7PPt733f5/k9+oir3hLdqIQsZaxqXTaihqAEyglvr9/kC4OTWErE5faCuNldrKI/LDDiIlchoz10WGv12PPKPM8xzjd2afsOB26ZTU9ov7/zyBW+2D7GKDC5Gghaz/2gxQPWFqsLXX5r/yGW7AE13RUGfFnkSU1jnX5cIsolClt+jY7vYCoRbuzw4e6DlFUxSTpqdbjvNzhmdYRnRxZ0qH23jCanlNSQUWhyOV5ktdRjyRgAMIkNEkWmpY25Nl2gobv0I5sn6vfpRmIU/7CzwaXJCh3JoReW+IG1L7MdiLG6q+jYSoib6owTk7rqFh0k51BCqE4xZBEiiwx1eSqKmlxOdYhZj1KFzaBOSQlzLLO4gdcllxcGawBo0quv6xqW8vDiNJP4dPs0upxgqhFlVUhG6prLhzsXCBNV6LTNHkr+3HEqszmqc7pxwNakxtVMoq57KFJGSQnYDmqie5mqDGObZb3HHX+WNbODJiV5x0wAJkwl4rjd4aXBKlGioKopG5O6yM3Kc31kKcOPxRZSNz1SJH5u763MmyOWKiMcLSBMFfq+IOCBgGMc5jT96Zkvciec5S2VW9wNBdzhureAowT0AlscaG7McPLBTX5n/wKylPG25m3+1e0nONrosRPV6Ecl3t28RpSpXG3P8b+98l5kOeO/v/BxPt47h6VE+IrGufIudXXK/MwQUw753f5DBKnK3qTMD689zx1/BkOOWTb67AQ1vuwvEacKDctlGJisVvrsuhUahsv6tEGUyZy295nEBjXdw09UTCVikNjMl0aU1YC277Ba6ovckljHkBP+m7UvkGQyXxie5MXhGpYicpRWrD5PNtcJUo2dsC6Kc1t4rQ7DlwF2ghpl1S+Q7cPYQpNzgmiqi5yWqFrsT0DhXXQUvyi+DqMfTElIXvuxzaw25oS5x/2wRS8HVQSShvQ6dRlZBm6ss+b0eKWzRNXwC3njvlemZU7YnIgQzb1pBVsLcfNpUpioTEKdc3N73OzOFEStOJWZxAYyGaqUEKY
```

Comment:
Can you update this to:

```python
change = (
    data.diff(dim="time")
        .assign_coords(time=["Winter", "Spring", "Summer"])
)
change.plot.imshow(cmap="BrBG", col="time")
```

Reply:
Thank you, I _knew_ there was a "proper" way to do this but couldn't figure it out :-).